### DISCUSSION TITLE
Angular specification with Storybooks addons

### DISCUSSION THREAD

**Discussion by RuBAN-GT** [storybookjs/storybook#11254] [Help] [answered]
Hello everyone 👋 

I have an Angular library that want to cover by tests with default Angular jasmine + karma stack.
From the Storybook page I found that React-world has https://github.com/mthuret/storybook-addon-specifications.  
Therefore I have a question: are there any stable storybook solution integrating between this framework and angular internal test runner?

Thank you in advance!

**Comment by shilman** [comment, marked as answer]
Probably best to ask on the `storybook-addon-specifications` repo if you haven't already.

That said, if you use [Component Story Format](https://medium.com/storybookjs/component-story-format-66f4c32366df), the recommendation since 5.2, you should be able to integrate Stories into ANY testing tool without extra work. There's a `jest` example in the post, but the same logic should be applicable in ALL modern testing environments.
